docs/quickstarts/nextjs.mdx

```diff
@@ -29,7 +53,11 @@ Once you have a Next.js application ready, you need to install Clerk's Next.js S
 
 ### Set environment keys
 
-Below is an example of an `.env.local` file. If you are signed into your Clerk Dashboard, your keys should become visible by clicking on the eye icon.
+In your Next.js project's root folder, you may have an `.env.local` file alongside `package.json` and other configuration files. If you don't see it, create it.

```

Comment:
"If you don't see it, create it" -- Love this cause I always find myself lost (sometimes we also have `.env.local.example`)

Reply:
I walked into this wall myself lol